cmd/ctr/commands/images/pull.go

```diff
@@ -65,9 +65,13 @@ command. As part of this process, we do the following:
 			Name:  "all-metadata",
 			Usage: "Pull metadata for all platforms",
 		},
+		cli.BoolFlag{
+			Name:  "no-unpack",
+			Usage: "skip unpacking the images, cannot be used with --print-chainid, false by default",
```

Comment:
Small nit: I would either format the default value as such ```... --print-chainid (default: false)``` to match other examples or drop the default value altogether to match other flag usage texts, but not required either.

Reply:
I copied this from the equivalent `ctr images import` flags, but I can reformat if a new rev is necessary

https://github.com/containerd/containerd/blob/main/cmd/ctr/commands/images/import.go#L207

Reply:
Ah, I see. Probably safe to ignore me then. :)

Reply:
`ctr i pull --help` shows the `(default: foo)` format for existing flags (well, just `--max-concurrent-downloads`), so I think it'd be reasonable to match that.

Reply:
Yeah, makes sense.